utils/set/set.go

```diff
@@ -23,6 +22,14 @@ var _ stdjson.Marshaler = (*Set[int])(nil)
 // Set is a set of elements.
 type Set[T comparable] map[T]struct{}
 
+// Of returns a Set initialized with [elts]
+func Of[T comparable](elts ...T) Set[T] {
+	s := NewSet[T](len(elts))
+	s.Add(elts...)
+
```

Comment:
super nit - but the space here looks out of place with the rest of the functions
```suggestion
```